.drone.starlark

```diff
@@ -0,0 +1,1326 @@
+config = {
+	'app': 'files_external_s3',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'appInstallCommand': 'composer install -n --no-progress',
+
+	'branches': [
+		'master'
+	],
+
+	'codestyle': True,
+
+	'phpstan': False,
+
+	'phan': False,
+
+	'phpunit': {
+		'scality': {
+			'databases': [
+				'sqlite',
+			],
+			'scalityS3': {
+				'createFirstBucket': False,
+				'extraEnvironment': {
+					'SCALITY_ACCESS_KEY_ID': 'owncloud123456',
+					'SCALITY_SECRET_ACCESS_KEY': 'secret123456'
+				}
+			},
+			'includeKeyInMatrixName': True,
+			'extraSetup': [
+				{
+					'name': 'scality-config',
+					'image': 'owncloudci/php:7.2',
+					'pull': 'always',
+					'commands': [
+						'cp tests/drone/configs/config.scality.php tests/unit/config.php'
+					]
+				}
+			],
+		},
+		'ceph': {
+			'databases': [
+				'sqlite',
+			],
+			'cephS3': True,
+			'includeKeyInMatrixName': True,
+			'extraSetup': [
+				{
+					'name': 'ceph-config',
+					'image': 'owncloudci/php:7.2',
+					'pull': 'always',
+					'commands': [
+						'cp tests/drone/configs/config.ceph.php tests/unit/config.php'
+					]
+				}
+			],
+		},
+	},
+
+	'acceptance': False,
+}
+
+def main(ctx):
```

Comment:
The starlark code from here is exactly the same as the code in `activity` app that has already been reviewed.